demos/simple/output.html

```diff
@@ -7,12 +7,12 @@
 <body>
     <div id="live-editor-output"></div>
 
-    <script src="../../build/js/live-editor.core_deps.js"></script>
+    <!-- TODO(kevinb7) use JSON dependency loading system for core_deps.js and shared.js -->
```

Comment:
Might be best to use whatever Perseus uses for their local demo, if we move to that here. Or at least check it out to see what that repo uses.